ir/v1.def

```diff
@@ -159,7 +159,12 @@ class NamedCond : If {
 
 class Apply : Expression {
     optional ID                         name;
+    // We should not use a NameMap, since it does not have source position information.
     inline NameMap<Vector<Expression>, ordered_map>  actions = {};
+    // We make up for it by storing the position information here.
+    inline NameMap<Path> position = {};
+    /*
+    // This is actually broken, since the srcInfo is 0 for all such vectors
```

Comment:
If it is broken maybe we should remove it instead of keeping it as commented-out code?